editor/src/components/canvas/controls/comment-mode/comment-mode-hooks.tsx

```diff
@@ -28,16 +50,47 @@ export function useCommentModeSelectAndHover(comment: CommentId | null): MouseCa
           storeRef.current.canvasOffset,
           windowPoint({ x: event.clientX, y: event.clientY }),
         )
+
+        const scenesUnderTheMouse = scenes.filter((scene) => {
+          const sceneId = getIdOfScene(scene)
+          return (
+            sceneId != null &&
+            isNotNullFiniteRectangle(scene.globalFrame) &&
+            rectContainsPoint(scene.globalFrame, loc.canvasPositionRaw)
+          )
+        })
+        const scene = scenesUnderTheMouse[0]
```

Comment:
it would be good to safely check for index 0

Reply:
done